templates/event/event_show.html.twig

```diff
@@ -1,118 +1,128 @@
-                {% for subgenre in event.subgenres%}
-                    {% if loop.last %}
-                        {{ subgenre.name }}
-                    {% else %}
-                        {{ subgenre.name }},
+                <h4 class="text-muted pl-1">
+                    {{ event.startDate | date("d/m/Y") }}
+                    {% if event.startDate != event.endDate %}
+                         - {{ event.endDate | date("d/m/Y") }}
                     {% endif %}
-                {% endfor %}
-            </p>
-        {% endif %}
-
-        {% if event.organizers %}
-            <p class="pl-1"> <strong> Организаторы:</strong> {{ event.organizers }} </p>
-        {% endif %}
-
-        {% if event.settlement %}
-            <p class="pl-1"> <strong> Тип поселения:</strong> {{ event.settlement.name }} </p>
-        {% endif %}
-
-        {% if not event.weapons is empty  %}
-            <p class="pl-1"> <strong> Материал оружия: </strong>
-                {% for weapon in event.weapons %}
-                    {% if loop.last %}
-                        {{ weapon.name }}
-                    {% else %}
-                        {{ weapon.name }},
+                </h4>
+                <h4 class="text-muted pl-1">
+                    {% if event.location %}
+                        {{ event.location }},
                     {% endif %}
-                {% endfor %}
-            </p>
-        {% endif %}
+                    {{ event.region.name }}
+                </h4>
 
-        {% if (event.priceMin or event.priceMax) %}
-            <p class="p-1">
-                <strong>Взнос: </strong>
-                {% if event.priceMin %}
-                    {{ event.priceMin }}
-                    {% if event.priceMax and event.priceMax != event.priceMin %}
-                        - {{ event.priceMax }}
-                    {% endif %}
-                {% else %}
-                    {{ event.priceMax }}
+                {% if event.type %}
+                    <p class="pl-1"> <strong> Формат:</strong> {{ event.type.name}} </p>
                 {% endif %}
-                 грн.
-            </p>
-        {% endif %}
 
-        {% if (event.contactSite or event.contactFB or event.contactVK or event.contactTelegram or event.contactOther) %}
-            <p class="p-1"> <strong> Ссылки: </strong>
-                {% if event.contactSite %}
-                    <a href="http://{{ event.contactSite }}" class="text-dark mx-3"> сайт </a>
+                {% if event.genre %}
+                    <p class="pl-1"> <strong> Жанр:</strong> {{ event.genre.name }} </p>
                 {% endif %}
-                {% if event.contactFB %}
-                    <a href="{{ event.contactFB }}" class="text-dark"> <span class="fa fa-facebook-square mx-3" ></span></a>
+
+                {% if event.subgenres %}
+                    <p class="pl-1"> <strong> Поджанры: </strong>
+                        {% for subgenre in event.subgenres%}
```

Comment:
@daria-sieroshtan 
This is a perfect place to do `{{ event.subgenres | join(', ') }}`
Notice that subgenre need `__toString()` method for this to work

Reply:
Won't "," be hanging as an orphan at the end in case of join?

Reply:
@daria-sieroshtan 
No, "glue" is inserted only inbetween, not in the ends
You can run `php -a` in concole and experiment with `implode()` (also known as `join()`)